scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -166,25 +165,30 @@ public static BuildTargetResult CheckPackageCache(BuildTargetContext c)
 
             if (ciBuild)
```

Comment:
If they aren't needed, let's remove them everywhere. This toggle was added on a whim later, but I didn't end up need it. I can remove it.